kratos/python_scripts/save_restart_process.py

```diff
@@ -17,19 +17,22 @@ def __init__(self, model, params):
         KratosMultiphysics.Process.__init__(self)
         ## Settings string in json format
         default_settings = KratosMultiphysics.Parameters("""{
-            "help"                         : "This process is used in order to save the problem databse with the serializer the current problem",
             "model_part_name"              : "SPECIFY_MODEL_PART_NAME",
             "echo_level"                   : 0,
             "serializer_trace"             : "no_trace",
             "restart_save_frequency"       : 0.0,
             "restart_control_type"         : "time",
             "save_restart_files_in_folder" : true,
-            "io_foldername"                : ""
+            "io_path"                : ""
```

Comment:
```suggestion
            "output_path"                : ""
```

this is what we started doing for the other processes

Reply:
My objection is: restart uses that path to output and input, thus the naming io. This is something that differentiates it from other processes, which only output data.

Reply:
this is the save_restart process, this one only outputs